build.sh

```diff
@@ -853,17 +853,17 @@ if [[ $__ClangMajorVersion == 0 && $__ClangMinorVersion == 0 ]]; then
             __ClangMinorVersion=6
         fi
 
-        if [[ "$__BuildArch" == "armel" ]]; then
-            # Armel cross build is Tizen specific and does not support Portable RID build
-            __PortableBuild=0
-        fi
-
     else
         __ClangMajorVersion=3
         __ClangMinorVersion=5
     fi
 fi
 
+if [[ "$__BuildArch" == "armel" ]]; then
```

Comment:
Yes, we can use that option explicitly.

However it seems that currently this option is turned off only when we don't use `clangX.Y` option and we want to make this option not dependent on `clangX.Y` option.

Furthermore  if you don't mind, it would be convenient to build coreclr with simple build command, i.e. `build.sh armel cross`, because this option is not supported for Tizen yet.